integration/cluster.go

```diff
@@ -711,6 +711,16 @@ func (m *member) StopWithAutoLeaderTransfer(t *testing.T) {
 	plog.Printf("stopped %s (%s)", m.Name, m.grpcAddr)
 }
 
+// checkLeaderTransition waits for leader transition, returning the new leader ID.
+func checkLeaderTransition(t *testing.T, m *member) uint64 {
+	oldLead := m.s.Lead()
+	interval := time.Duration(m.s.Cfg.TickMs) * time.Millisecond
+	for m.s.Lead() == 0 || (m.s.Lead() == oldLead) {
```

Comment:
why would `Lead() == 0`, doesn't that mean there's an election in progress? shouldn't the leader be either the old leader or the new one?